scripts/update-guardian-set.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+# This script submits a guardian set update using the VAA injection admin command.
+# First argument is the number of guardians for the new guardian set.
+set -e
+
+# wait for the guardians to establish networking
+sleep 30
+
+newNumGuardians=$1
+echo "new number of guardians: ${newNumGuardians}"
+
+webHost=$2
+echo "webHost ${webHost}"
+
+# file & path to save governance VAA
+fileName=new-guardianset.prototxt
+localPath=./scripts/$fileName
+containerPath=/tmp/$fileName
+echo "localPath: ${localPath}"
+echo "containerPath: ${containerPath}"
+
+sock=/tmp/admin.sock
+
+guardianPublicWebBaseUrl="${webHost}:7071"
+
+currentGuardianSetUrl="${guardianPublicWebBaseUrl}/v1/guardianset/current"
+
+# fetch result and parse json body:
+guardianSet=$(curl ${currentGuardianSetUrl} | jq ".guardianSet")
+currentIndex=$(echo ${guardianSet} | jq ".index")
+currentNumGuardians=$(echo ${guardianSet} | jq ".addresses | length")
+echo "currentIndex: ${currentIndex}"
+echo "currentNumGuardians ${currentNumGuardians}"
+
+echo "creating guardian set update governance message template prototext"
+minikube kubectl -- exec -n wormhole guardian-0 -c guardiand -- /guardiand template guardian-set-update --num=${newNumGuardians} --idx=${currentIndex} > ${localPath}
+
+# for i in $(seq ${newNumGuardians})
+for i in $(seq ${currentNumGuardians})
+do
+  # create guardian index: [0-18]
+  guardianIndex=$((i-1))
+
+  # create the governance guardian set update prototxt file in the container
+  echo "created governance file for guardian-${guardianIndex}"
+  minikube kubectl -- cp ${localPath} wormhole/guardian-${guardianIndex}:${containerPath} -c guardiand
+
+  # inject the guardian set update
+  minikube kubectl -- exec -n wormhole guardian-${guardianIndex} -c guardiand -- /guardiand admin governance-vaa-inject --socket $sock $containerPath
+  echo "injected governance VAA for guardian-${guardianIndex}"
+done
+
+# wait for the guardians to reach quorum about the new guardian set
+sleep 30 # probably overkill, but some waiting is required.
```

Comment:
Is there a better way to do this event based? This PR adds up to a full minute so far of extra time to the tests